tests/auth0/validator.tests.js

```diff
@@ -4,7 +4,22 @@ import Auth0 from '../../src/auth0';
 describe('#schema validation tests', () => {
   const client = {
     rules: {
-      getAll: () => []
+      getAll: async () => ({ rules: [] })
```

Comment:
We only had it for `rules`, this makes all the tests to hit the proxy instead of just rules.